sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/command.py

```diff
@@ -240,6 +241,29 @@ def resources(self, value: Union[Dict, JobResourceConfiguration]):
             value = JobResourceConfiguration(**value)
         self._resources = value
 
+    @property
+    def identity(
+        self,
+    ) -> Optional[Union[ManagedIdentityConfiguration, AmlTokenConfiguration, UserIdentityConfiguration]]:
+        return self._identity
+        
+    @identity.setter
+    def identity(self, value: Union[
```

Comment:
need to add some test for the new capability

Reply:
added in dsl/unittests/test_command_builder.py